src/storage/v3/shard.hpp

```diff
@@ -368,7 +391,7 @@ class Shard final {
   // Main object storage
   NameIdMapper name_id_mapper_;
   LabelId primary_label_;
-  // The shard's range is [min, max)
+  // The shard's range is [min, max>
```

Comment:
```suggestion
  // The shard's range is [min, max)
```

This should remain unchanged, as it is the correct notation for a half-open interval: https://en.wikipedia.org/wiki/Interval_%28mathematics%29#Terminology